lib/checks/canFindRoots.js

```diff
@@ -0,0 +1,35 @@
+const Node = require('../node');
+const resolvesToConstant = require('./resolvesToConstant.js');
+
+function canFindRoots(equation) {
+
+  // Equation should be in the form of factor * factor = 0, factor^power = 0, or identifier = 0
+  // e.g (x - 2)^2 = 0, x(x + 2)(x - 2) = 0
+  const left = equation.leftNode;
+  const right = equation.rightNode;
+
+  const zeroRightSide = Node.Type.isConstant(right)
```

Comment:
@aliang8 I think @evykassirer wanted some sort of helper function here, do you remember?